provisioning/resources/control-plane/scripts/template_iglu_server

```diff
@@ -0,0 +1,13 @@
+      {
+        "name": "Iglu Server",
+        "priority": 0,
+        "vendorPrefixes": [
+          "com.snowplowanalytics"
```

Comment:
How does this template handle other vendors?  Remember that people outside of snowplow will have a very different vendor prefix.